.golangci.yaml

```diff
@@ -76,24 +76,28 @@ linters-settings:
       #
       # Because it's easier to read without the other fields.
       #
-      - 'GetPayloadsFilters'
+      - "GetPayloadsFilters"
 
       #
       # Structures outside our control that have a ton of settings. It doesn't
       # make sense to specify all of the fields.
       #
-      - 'cobra.Command'
-      - 'database.*Entry'
-      - 'http.Server'
-      - 'logrus.*Formatter'
-      - 'Options' # redis
+      - "cobra.Command"
+      - "database.*Entry"
+      - "http.Server"
+      - "logrus.*Formatter"
+      - "Options" # redis
 
       #
       # Excluded because there are private fields (not capitalized) that are
       # not initialized. If possible, I think these should be altered.
       #
-      - 'Datastore'
-      - 'Housekeeper'
-      - 'MockBeaconClient'
-      - 'RelayAPI'
-      - 'Webserver'
+      - "Datastore"
+      - "Housekeeper"
+      - "MockBeaconClient"
+      - "RelayAPI"
+      - "Webserver"
+
+      #
+      # Optional fields for the prometheus metrics
+      - "GaugeOpts"
```

Comment:
This is the only change here, the rest is just changing from single into double quotes  applied from the auto formatting from vscode for yuml